cmd/ask_pass_test.go

```diff
@@ -56,6 +56,7 @@ func NewTestCommand() *cobra.Command {
 			fmt.Fprintf(c.ErrOrStderr(), "%s is not set\n", git.ASKPASS_NONCE_ENV)
 			return
 		}
+		//nolint
```

Comment:
Is this required for this issue? It seems lint has been running fine without this, and we can probably revert the above line.

Reply:
I had problems without this : SA1019: grpc.DialContext is deprecated: use NewClient instead.
Probably linked with the go upgrade